browser-test/src/program_deep_link.test.ts

```diff
@@ -96,4 +101,25 @@ describe('navigating to a deep link', () => {
       questionText,
     )
   })
+
+  it('Non-logged in user should get redirected to the program page and not an error', async () => {
+    await resetContext(ctx)
+    const {page, browserContext} = ctx
+
+    await selectApplicantLanguage(page, 'English')
+
+    await logout(page)
+    await browserContext.clearCookies()
```

Comment:
if `clearCookies()` accomplishes what `resetContext()` was meant to do here, then maybe it's worth simplifying the changes in `index.ts`

Reply:
I tried putting resetContext back and it did not pass. The call to clearCookies was just meant to be double certain there were no lingering auth cookies around. I can probably remove it as the tests still pass without.